Test/RivetTest/Functions/Start-RivetTest.ps1

```diff
@@ -68,7 +70,8 @@ function Start-RivetTest
     @"
 {
     SqlServerName: '$($RTServer.Replace('\', '\\'))',
-    DatabasesRoot: '$($RTDatabasesRoot.Replace('\','\\'))'
+    DatabasesRoot: '$($RTDatabasesRoot.Replace('\','\\'))',
+    DatabaseOrder: [$DatabaseOrder]

```

Comment:
Rename this to just `Databases`. Modify `Get-RivetConfig` so that if this property isn't present, it grabs all databases from the databases root. If this property **is** present, **only** databases that match an entry in this list is returned, i.e. loop through Databases and `Get-Item -Path $databasesEntry` for each entry. 

Add some tests for this new behavior:

* only databases in this list are returned
* if using `*` as the last entry in the list, no duplicate databases are returned and all explicit database name before the `*` in the list stay in that order

I.e. users may want to do something like:

```json
    "Databases": [
        "Common",
        "WellMed",
         "*"
    ]
``` 

Also, only output the `Databases` property if someone passes in a `ConfigurationDatabase` argument.